tailwind.config.ts

```diff
@@ -111,6 +111,7 @@ export default {
     },
     fontFamily: {
       'open-sans': ['var(--font-open-sans)'],
+      mono: ['var(--font-ibm-plex-mono)'],
```

Comment:
This should be called 'ibm-plex-mono'

```suggestion
      'ibm-plex-mono': ['var(--font-ibm-plex-mono)'],
```